go.sum

```diff
@@ -432,6 +432,8 @@ github.com/vmihailenco/msgpack v4.0.1+incompatible/go.mod h1:fy3FlTQTDXWkZ7Bh6Ac
 github.com/xanzy/ssh-agent v0.2.1/go.mod h1:mLlQY/MoOhWBj+gOGMQkOeiEvkx+8pJSI+0Bx9h2kr4=
 github.com/xaptum/go-enf v0.1.0 h1:SaLOPP3473pELe9uFkSigAOJcRjTRxXnOx2/3jxXgTY=
 github.com/xaptum/go-enf v0.1.0/go.mod h1:mlduHzSTPwpuoYtNbJvAwqFjbHZdFu4p0NSLiWpzUx0=
+github.com/xaptum/go-enf v0.2.0 h1:858BJ723XAtUGP2/aaTpVNIZsu2WnATjIRuhj7ELXWU=
```

Comment:
The `enf-go` version didn't get updated in `go.sum`.  Maybe you need to `make build` and then commit the modified `go.sum`?

Reply:
Gotcha. I'll do that and fixup the commit real quick.